js/index.js

```diff
@@ -1015,31 +993,32 @@ async function getModels() {
 		console.warn(e);
 	}
 
-	modelAutoComplete.onchange.on(async ({value}) => {
-		console.log(`[index] Changing model to [${value}]`);
-		const payload = {
-			sd_model_checkpoint: value,
-		};
-		const url = document.getElementById("host").value + "/sdapi/v1/options/";
-		try {
-			await fetch(url, {
-				method: "POST",
-				headers: {
-					"Content-Type": "application/json",
-				},
-				body: JSON.stringify(payload),
-			});
-
-			alert(`Model changed to [${value}]`);
-		} catch (e) {
-			console.warn("[index] Error changing model");
-			console.warn(e);
+	if (!refresh)
+		modelAutoComplete.onchange.on(async ({value}) => {
+			console.log(`[index] Changing model to [${value}]`);
+			const payload = {
+				sd_model_checkpoint: value,
+			};
+			const url = document.getElementById("host").value + "/sdapi/v1/options/";
+			try {
+				await fetch(url, {
+					method: "POST",
+					headers: {
+						"Content-Type": "application/json",
+					},
+					body: JSON.stringify(payload),
+				});
+
+				alert(`Model changed to [${value}]`);
+			} catch (e) {
+				console.warn("[index] Error changing model");
+				console.warn(e);
```

Comment:
i am genuinely angry at myself because i tried LITERALLY EXACTLY THIS VERBATIM like 4 or 5 times, except for the changes in ui.js -_-

Reply:
Yeah, ui.js had some tiny bugs on the set options haha had to fix those!